README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 3.2.0
```

Comment:
Consider using semantic versioning properly by incrementing the version number more meaningfully, such as 3.3.0 instead of 3.2.0.